ntbs-service/Services/CultureAndResistanceService.cs

```diff
@@ -0,0 +1,56 @@
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Dapper;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+using ntbs_service.Models.Entities;
+
+namespace ntbs_service.Services
+{
+    public interface ICultureAndResistanceService
+    {
+        Task<CultureAndResistance> GetCultureAndResistanceDetailsAsync(int notificationId);
+    }
+
+    public class CultureAndResistanceService : ICultureAndResistanceService
+    {
+        private readonly string connectionString;
+
+        public CultureAndResistanceService(IConfiguration _configuration)
+        {
+            connectionString = _configuration.GetConnectionString("reporting");
+        }
+
+        public async Task<CultureAndResistance> GetCultureAndResistanceDetailsAsync(int notificationId)
+        {
+            string query = "SELECT * FROM [dbo].[ufnGetNotificationCultureAndResistanceSummary] (@notificationId)";

```

Comment:
Would prefer this query to be explicit with which fields are returned

Reply:
that will be lots of fields, I am not sure they will be much useful as they are same as CultureAndResistance properties

Reply:
13 isn't too crazy - plus a bit of string-bloat to have a restricted query is worth it IMO.
Would also give a more explicit error if down the line anything changed on their end which impacted on this functionality.